templates/chrony.conf.j2

```diff
@@ -79,6 +79,10 @@ ntsdumpdir /var/lib/chrony
 # Limit maximum root distance.
 maxdistance {{ timesync_max_distance }}
 
+{% if __timesync_chrony_version is version('4.0', '>=') %}
```

Comment:
```suggestion
{% if __timesync_chrony_version is version('4.0', '>=') and timesync_chrony_leapsectz %}
```

Reply:
I would prefer not to add a new setting by default - I would prefer to say "if leapsectz isn't set, use the default built into chronyd"

Reply:
Also note that using `| default(...)` here isn't necessary - since it is defined in defaults/main.yml, it will *always* be defined - the only thing to check is to evaluate it in a boolean context to see if it is set to something other than the empty string `""`